app/styles/organisms/sidebar.less

```diff
@@ -0,0 +1,128 @@
+@media screen and (-webkit-device-pixel-ratio: 1) {
+  .oss-sidebar-item--locked {
+    padding-left: 1px;
+    padding-bottom: 1px;
+  }
+
+  .oss-sidebar-item--icon {
+    padding: 1px;
+  }
+}
+
+.oss-sidebar--containers {
+  position: fixed;
+  height: 100vh;
+  width: var(--sidebar-width);
+
+  background-color: var(--sidebar-bg-color);
+  border-right: 1px solid var(--sidebar-outer-border-color);
+  padding: var(--spacing-px-12) var(--spacing-px-9);
+
+  .logo-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    min-height: var(--topbar-height);
+    max-height: var(--topbar-height);
+
+    margin-right: -1px;
```

Comment:
is this to align the border or something ? 🤔

Reply:
This is it! Without it there is a missing pixel at the end of the inner border